VitalTrackApp/src/screens/Login.js

```diff
@@ -1,77 +1,47 @@
-import auth from '@react-native-firebase/auth';
-import {useNavigation} from '@react-navigation/native';
-import React, {useState} from 'react';
 import {
-  Image,
-  StyleSheet,
+  View,
   Text,
   TextInput,
   TouchableOpacity,
-  View,
+  StyleSheet,
+  Image,
 } from 'react-native';
-
+import { useNavigation } from '@react-navigation/native'; // Import useNavigation
+// test
 export default function LoginScreen() {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const navigation = useNavigation();
-
-  function handleSubmit() {
-    loginUser(email, password);
-  }
-
-  async function loginUser(email, password) {
```

Comment:
why did we just remove all functionality???

Reply:
I updated the file so it reflects current changes now